### PR TITLE
Nirali

### PR DESCRIPTION
**Pull request by nira24**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   database:
-    image: postgres:13-alpine
+    image: "postgres:13-alpine"
```
Comment (marcelala) [inline docker-compose.yaml:4]: is this ok?

Comment (marcelala) [reply docker-compose.yaml:4]: mine doesn't have ""

Comment (nira24) [reply docker-compose.yaml:4]: ya its fine shouldn't be a problem

**Review by marcelala — APPROVED** [review]
looks good as far as I can follow 👍

**Pull request merged**